iothub/service/src/DirectMethod/DirectMethodsClient.cs

```diff
@@ -41,10 +43,10 @@ internal DirectMethodsClient(string hostName, IotHubConnectionProperties credent
         /// Invokes a method on a device.
         /// </summary>
         /// <param name="deviceId">The device identifier for the target device.</param>
-        /// <param name="cloudToDeviceMethod">Parameters to execute a direct method on the device.</param>
+        /// <param name="directMethodRequest">Parameters to execute a direct method on the device.</param>
         /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
-        /// <returns>The <see cref="CloudToDeviceMethodResult"/>.</returns>
-        /// <exception cref="ArgumentNullException">Thrown when the provided <paramref name="deviceId"/> or <paramref name="cloudToDeviceMethod"/> is null.</exception>
+        /// <returns>The <see cref="DirectMethodResponse"/>.</returns>
```

Comment:
Don't link to the type. It is already present in the API surface and will be automatically linked.